scripts/macosx-build-dependencies.sh

```diff
@@ -53,6 +53,7 @@ PACKAGES=(
     "libxml2 2.9.7"
     "fontconfig 2.12.4"
     "hidapi 0.7.0"
+    "lib3mf a466df47231c02298dde295adb6075b0fc649eba"
```

Comment:
I see there's a bunch of recent activity on lib3mf - should we bump this to a more recent version?